src/lib/Auth.svelte

```diff
@@ -26,13 +26,48 @@
       loadingSignIn = false
     }
   }
+
+  const handleSignInWithOAuth = async () => {
+    try {
+      loadingSignIn = true
+
+      const { data, error } = await supabase.auth.signInWithOAuth({
+        provider: 'github'
+      })
+
+      if (error) throw error
+      console.log(`Logged in user: ${JSON.stringify(data, undefined, 2)}`)
+    } catch (error) {
+      console.error(error)
+    } finally {
+      loadingSignIn = false
+    }
+  }
 </script>
 
 <div class="row flex-center flex">
   <div class="col-6 form-widget" aria-live="polite">
     <h1 class="header">Hacktoberfest Tracker</h1>
+    <p class="description">Sign in using your GitHub account</p>
```

Comment:
I think this is sort of redundant - I'd lean towards just having the button which speaks for itself. But it's not a big deal - some of this will change once I add in a component library anyway